docs/pages/bare/updating-your-app.md

```diff
@@ -77,17 +77,21 @@ Since headers sent in requests by `expo-updates` do not affect statically hosted
 
-Updates embedded into your app binary look slightly different from how vanilla React Native apps embed JavaScript and assets. In particular, updates from `expo-updates` include the manifest metadata along with all of the assets required to run the update. The manifest should be in a file called `app.manifest` and the JavaScript bundle in a file called `app.bundle`. For iOS, these should be added to your Xcode project so they are bundled into the IPA. For Android, placing them in the `android/app/src/main/assets` folder is sufficient for gradle to bundle them into the APK.
+When you make a release build of your app, the build process will bundle your JavaScript source code into a minifed bundle and embed this in the binary, along with any other assets your app `require`s. `expo-updates` includes an extra script on each platform to embed some additional metadata about the embedded assets -- namely, a minimal manifest JSON object for the update.
 
-Assets that you `require` in your JavaScript source can be embedded into your app binary. If you use `expo-asset` in your project (included by default if you have the `expo` package installed), you can control which `require`d assets will be bundled into your binary using the [assetBundlePatterns](../../workflow/configuration/) key in `app.json` to provide a list of paths in your project directory:
+## Including Assets in Updates
+
+Assets that you `require` in your JavaScript source can also be atomically downloaded as part of a published update. `expo-updates` will not consider an update "ready" and will not launch the update unless it has downloaded all required assets.
+
+If you use `expo-asset` in your project (included by default if you have the `expo` package installed), you can control which `require`d assets will be included as part of this atomic update by using the [assetBundlePatterns](../../workflow/configuration/) key in `app.json` to provide a list of paths in your project directory:
 
 ```
 "assetBundlePatterns": [
   "**/*" // or "assets/images/*", etc.
 ],
 ```
 
-Assets with paths matching the given patterns will be bundled into your native binaries next time you run `expo build`. If you have an asset that should be lazily downloaded at runtime rather than when the app is installed, you can use `assetBundlePatterns` to exclude it while still `require`ing it in your JavaScript code.
+Assets with paths matching the given patterns will be pre-downloaded by clients before the update that uses them will launch. If you have an asset that should be lazily downloaded at runtime rather than before your JavaScript is evaluated, you can use `assetBundlePatterns` to exclude it while still `require`ing it in your JavaScript source.
```

Comment:
```suggestion
Assets with paths matching the given patterns will be pre-downloaded by clients before the update that uses them will launch. If you have an asset that should be lazily downloaded at runtime rather than before your JavaScript is evaluated, you can use `assetBundlePatterns` to exclude it while still importing it in your JavaScript source.
```